packages/wrangler/src/index.tsx

```diff
@@ -260,9 +260,17 @@ export async function main(argv: string[]): Promise<void> {
             JSON.stringify(
               {
                 compilerOptions: {
-                  target: "ES2020",
-                  module: "CommonJS",
-                  lib: ["ES2020"],
+                  target: "esnext",
+                  module: "esnext",
+                  moduleResolution: "node",
+                  esModuleInterop: true,
+                  allowJs: true,
+                  allowSyntheticDefaultImports: true,
+                  isolatedModules: true,
```

Comment:
Maaaybe we should throw an error if esbuild is asked to compile a `.ts` file but `isolatedModules` is set to `false`. Maybe. Kicking it down the road...

Reply:
Does eslint not error on our behalf?

Reply:
looks like esbuild doesn't error on this.